packages/amplify-nodejs-function-template-provider/resources/lambda/crud/app.js.ejs

```diff
@@ -81,21 +81,20 @@ app.get(path + hashKeyPath, function(req, res) {
     KeyConditions: condition
   }
 
-  dynamodb.query(queryParams, (err, data) => {
-    if (err) {
-      res.statusCode = 500;
-      res.json({error: 'Could not load items: ' + err});
-    } else {
-      res.json(data.Items);
-    }
-  });
+  try {
+    const data = await ddbDocClient.send(new QueryCommand(queryParams));
+    res.json(data.Items);
+  } catch (err) {
+    res.statusCode = 500;
+    res.json({error: 'Could not load items: ' + err});
```

Comment:
Should this be `+ err` or `+ err.message`?

Line 133 is different.

Reply:
That came from old promise based code without change. But it seems err.message is more appropriate here. I'll change.

Reply:
updated.